packages/sonarwhal/src/lib/utils/npm.ts

```diff
@@ -12,21 +12,43 @@ import * as logger from './logging';
 const debug: debug.IDebugger = d(__filename);
 const npmLoadAsync = promisify(npm.load);
 
-
-const packageExists = () => {
+const getPackageContent = (): string => {
```

Comment:
Shouldn't this go into `misc` instead of `npm`? We have a few methods about `package.json` in there already (for finding, etc.). Maybe you can use `misc.findPackageRoot()` and check if the returned path is the same as `process.cwd()`?